h/views/api_profile.py

```diff
@@ -0,0 +1,11 @@
+from pyramid import security
```

Comment:
New Python files should always start with this magic comment: `# -*- coding: utf-8 -*-`

And to aid the efforts of moving our code base to Python 3, I usually also add: `from __future__ import unicode_literals`.